GameData/002_AdditionalConfigs/Knes.cfg

```diff
@@ -812,4 +975,46 @@
     @description ^=:$: <b><color=green>From  mod</color></b>
     %MODULE[ModuleTagList] { tag = NoResourceCostMult }
 	%identicalParts = _Knes_mrk_SM_Extended,_Knes_mrk_Pod,_Knes_mrk_Parachute
+}
+
+// Entry cost modifiers
+
+@ENTRYCOSTMODS:FOR[RP-0]{
+
+	//ESA costing
+	Hermes = 1150000
+	ESA-mk1-capsule = ESA-mk1-LES, 75000
+	ESA-MRK = 650000
+	ESA-MRK-LES = 150000
+	ESA-mk1-LES = 25000
+
+}
+
+@ENTRYCOSTMODS:FOR[xxxRP-0]{
+
+	//MRK parts
```

Comment:
All underscores in part names have to be replaced with dashes '-' or the modifier won't be applied.